remappings.txt

```diff
@@ -1,2 +1,3 @@
 @openzeppelin/contracts/=dependencies/@openzeppelin-contracts-5.4.0/
+bls-solidity/=dependencies/bls-solidity-0.1.0/src/libraries/
```

Comment:
Looks like my comment was lost: this seems like an interesting remapping to do since we might have code in a different subdirectory than `libraries/`, but we don't have to address this now @najienka

Reply:
Good point. We can always change the path if we need to consume more code in a different directory.